### PR TITLE
feat(header/locale): add icon and adjust

### PR DESCRIPTION
**Pull request by SivanYeh**
## Types of changes

* **New feature**

## Description

Add globe icon and adjust language name for language switch on header.

## Steps to Test This Pull Request

1. Go to "http://localhost:3000/2024/zh-hant"
2. Check if new icon is added
3. Check if language name is consistent

## Related Issue

#520 

## Screenshot

image
image

### CODE REVIEW

**Comment by tyuchx** [comment]
LGTM

**Comment by SivanYeh** [comment]
> LGTM

Can you approve this PR for me?

**Comment by gtbcoding** [comment]
LGTM

**Review by tyuchx — APPROVED** [review]
LGTM

**Review by gtbcoding — APPROVED** [review]
LGTM

File: components/core/header/locale-switch/LocaleSwitch.i18n.js
Comment (mattwang44) [inline components/core/header/locale-switch/LocaleSwitch.i18n.js:1]: 既然一樣就可以不透過  i18n.js 拿資料

**Pull request merged**